src/tradingtimes/tradingTimes.es6

```diff
@@ -146,6 +146,7 @@ const initTradingWin = ($html) => {
         const result = processData(menu.extractFilteredMarkets(data[1]));
         const header = getMarketsSubmarkets(data[0].active_symbols);
         const market_name_list = Object.keys(header);
+        const market_name_reversed = market_name_list.length > 1 ? Object.assign([], market_name_list, {[2]: market_name_list[3], [3]: market_name_list[2]}) : market_name_list; // handle norway clients (handle translation)
```

Comment:
and use the function here too (to keep things dry 😄 )

Reply:
indeed